### DISCUSSION TITLE
IntelliJ - login infinite loop

### DISCUSSION THREAD

**Discussion by junaexp** [community/community#41072] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

Login is disabled every 10-30 minutes.
github uses two-factor authentication.
If click "login to github", it will be used immediately.

![2022-12-08 18 47 02](https://user-images.githubusercontent.com/21023621/206414731-57c3d328-98f9-4b4c-834a-0943d0c577b7.png)

**Comment by Akash1134** [comment, marked as answer]
Hi @junaexp , we're sorry that you're facing problems using Copilot. Few questions:

- Did you use SSH to connect to Github? Also, please check that IntelliJ is set to save your password.
- Have you tried removing account altogether and then adding it again?
- Check that your Github account is associated with the IDE (IntelliJ) in use (Settings -> Version control -> Github)
- What OS did you use?

**Reply by junaexp** [reply]
Thank you for answer :)

problem is My VPN -_-a
Thanks for pointing out another workaround !
Be always happy hacking 👍
